mercury/templates/dashboard.html

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8"  http-equiv="refresh" content="5; URL=/dashboard/"/>
+    <meta charset="UTF-8">
+{#    http-equiv="refresh" content="5; URL=/dashboard/"/>#}
```

Comment:
Just want to make sure you intended to leave this commented out.

Reply:
yes